_topic_maps/_topic_map.yml

```diff
@@ -1998,8 +1998,8 @@ Name: Monitoring
 Dir: monitoring
 Distros: openshift-enterprise,openshift-origin
 Topics:
-- Name: Understanding the monitoring stack
-  File: understanding-the-monitoring-stack
+- Name: Overview of monitoring
```

Comment:
I hate to bring this up, given the amount of changes. But we have many books that use the pattern **BlaBlaBla overview**, rather than **Overview of BlaBlaBla**. See Architecture, Installing, Support, and so on.

Whatever the merits of one construction over another, we should always aim for consistency wherever possible.